GeneralsMD/Code/GameEngine/Source/GameClient/SelectionInfo.cpp

```diff
@@ -385,12 +385,6 @@ Bool addDrawableToList( Drawable *draw, void *userData )
 			{
 				return FALSE;
```

Comment:
I suspect this code here is fixing what we refer to in Generals as "box trick" aka selecting objects in the fog?